packages/datadog-instrumentations/src/google-cloud-pubsub.js

```diff
@@ -0,0 +1,122 @@
+'use strict'
+
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const requestStartCh = channel('apm:google-cloud-pubsub:request:start')
+const requestInjectMsgCh = channel(`apm:google-cloud-pubsub:request:injectMsg`)
+const receiveStartCh = channel(`apm:google-cloud-pubsub:receive:start`)
+
+const requestAsyncEndCh = channel('apm:google-cloud-pubsub:request:finish')
```

Comment:
The variable should have the same name as the channel to make things more readable.